frontend/src/components/layout/sidebar.module.css

```
@@ -409,8 +430,16 @@
     display: none;
   }
 
+  /* .ht-layout drops its padding here, so the drawer gets the full viewport back. */
   :global(.ht-drawer) .sidebar {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)